wave_gtsam/src/preint_imu_factor.cpp

```diff
@@ -0,0 +1,198 @@
+#include "wave/gtsam/preint_imu_factor.hpp"
```

Comment:
general thought, should we start using a wavelab header again? Would look nice.

Can be a later change haha